backend/endpoints/portfolio/route.py

```diff
@@ -0,0 +1,41 @@
+from typing import List
+from ninja import Router, Schema
+from lib.polygon_api import PolygonAPI
+
+from endpoints.auth import AuthBearer, AuthenticatedRequest
+from models.models import HoldingModel, UserModel
+
+router = Router(auth=AuthBearer())
+
+
+class StatsResponseSchema(Schema):
+    current_value: float
+
+
+@router.get("/stats", response=StatsResponseSchema)
+def stats(request: AuthenticatedRequest) -> StatsResponseSchema:
+    user = UserModel.objects.get(username=request.auth)
+    holdings = HoldingModel.objects.filter(user=user)
+
+    api = PolygonAPI()
+
+    current_value = sum([api.recent_price(holding.stock.ticker).low * float(holding.units)
+                        for holding in holdings])
+
+    return StatsResponseSchema(current_value=current_value)
+
+
+class HoldingSchema(Schema):
+    ticker: str
+    units: float
+
+    @staticmethod
+    def from_model(model: HoldingModel) -> 'HoldingSchema':
+        return HoldingSchema(ticker=model.stock.ticker, units=float(model.units))
```

Comment:
I'd suggest adding a field here to record the total value of the holding when acquired, so that it's possible to calculate overall gain/loss :)

Reply:
The holding isn't just acquired in one go - it's added to and sold at different points, so it's difficult to pin down exactly what the original price was. Potentially could do something like total value when bought, and then decrease in proportion to units sold when they sell?

Reply:
Yeah I agree that it's kinda difficult to pin down - so I'd suggest doing this. With the trading sequence you just posted:
```
units += 2
acquiredValue += 2*5

units += 1
acquiredValue += 10

units -= 1
acquiredValue -= 15
```
So at the end we'd have `acquiredValue = 5` and `units = 2`, and assuming that's everything in the portfolio the total gain would be `currentValue - acquiredValue = 10 * 2 - 5`. 

Feel like this could potentially work?

Reply:
That wouldn't work - consider the following situation:
 - buy 2 @ 5, price is now 10
 - sell 1 @ 10, price still 10
 - sell 1 @ 10, price still 10

In your model, the unrealised gain after each step would be: 
 - 20 - 10 = 10
 - 10 - (2*5 - 10) = 10
 - 0 - (2*5 - 10 - 10) = 10

The linear decrease model works more intuitively - as items are sold, the unrealised gain converges to 0. It seems like the method is called "average cost basis" online. I've implemented a version that seems to work well enough.

Reply:
See latest commit for implementation.